doc/source/python-tiledbsoma.rst

```diff
@@ -0,0 +1,52 @@
+The tiledbsoma module
+=====================
+
+.. currentmodule:: tiledbsoma
+
+.. automodule:: tiledbsoma
+
+Classes
+-------
+
+.. autosummary::
+    :toctree: _autosummary/
+    :nosignatures:
+
+    tiledbsoma.AxisColumnNames
+    tiledbsoma.AxisQuery
+    tiledbsoma.Collection
+    tiledbsoma.DataFrame
+    tiledbsoma.DenseNDArray
+    tiledbsoma.Experiment
+    tiledbsoma.ExperimentAxisQuery
+    tiledbsoma.Measurement   
+    tiledbsoma.ResultOrder
+    tiledbsoma.SOMATileDBContext
+    tiledbsoma.SparseNDArray
```

Comment:
```suggestion
    tiledbsoma.Collection
    tiledbsoma.Experiment
    tiledbsoma.Measurement   

    tiledbsoma.DataFrame
    tiledbsoma.SparseNDArray
    tiledbsoma.DenseNDArray

      tiledbsoma.ResultOrder

    tiledbsoma.AxisColumnNames
    tiledbsoma.AxisQuery
    tiledbsoma.ExperimentAxisQuery
    
    tiledbsoma.SOMATileDBContext
```

the first three are collections

the next three are foundational types

result order is a helper for use by the foundational types' `read`

the next three are for queries

that last one is common to all